Lib/test/test_math.py

```diff
@@ -1214,7 +1214,7 @@ def testPow(self):
         self.assertRaises(ValueError, math.pow, 0., -2.)
         self.assertRaises(ValueError, math.pow, 0., -2.3)
         self.assertRaises(ValueError, math.pow, 0., -3.)
-        self.assertRaises(ValueError, math.pow, 0., NINF)
+        self.assertEqual(math.pow(0., NINF), INF)
```

Comment:
For the consistent result check, 
I would like to suggest adding below tests

````python
self.assertEqual(math.pow(0., NINF), 0. ** NINF)
self.assertEqual(math.pow(-0., NINF), (-0.) ** NINF)
````

Reply:
Ah, I read the following comment.
https://github.com/python/cpython/blob/40a928429d7f3951c0a593f39d1b26f7e02a7529/Lib/test/test_math.py#L1330

Please ignore my comment.